### DISCUSSION TITLE
Shoko Server

### DISCUSSION THREAD

**Discussion by shyonae** [selfhst/icons#2] [New Icon Request]
Hi! Could we get Shoko Server's icon please? 

The project is this one: https://github.com/ShokoAnime/ShokoServer

And the icon is already present [here](https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons/png/shoko.png).

![shoko](https://github.com/user-attachments/assets/59f95935-fa70-4a45-99ad-65573155e593)

**Comment by shollyethan** [comment]
This has been added. Bonus points if anyone has access to the SVG that they wouldn't mind sharing!

**Reply by shyonae** [reply]
Should be present [here](https://github.com/walkxcode/Dashboard-Icons/blob/main/svg%2Fshoko.svg), thanks!
